examples/Training/python/Task_Scripts/Raw_Data2.py

```diff
@@ -98,6 +83,8 @@
 image.linkAnnotation(fileAnn)
 
 message = "Tile average value: %s" % average
-# client.setOutput("Message", rstring(message))
-# client.setOutput("Kymograph", robject(fileAnn._obj))
-# client.closeSession()
+
+# Close connection:
+# =================================================================
+# When you are done, close the session to free up server resources.
+conn._closeSession()
```

Comment:
no need to use privet method, see https://github.com/openmicroscopy/openmicroscopy/pull/4803